.github/workflows/ci.yml

```diff
@@ -48,12 +48,9 @@ jobs:
           sudo mv nextflow /usr/local/bin/
 
       - name: Run pipeline with test data
-        # TODO nf-core: You can customise CI pipeline run tests as required
-        # For example: adding multiple test runs with different parameters
-        # Remember that you can parallelise this by using strategy.matrix
         run: |
           nextflow run ${GITHUB_WORKSPACE} -profile test,docker
 
       - name: Run pipeline with additional params
         run: |
-          nextflow run ${GITHUB_WORKSPACE} -profile test,docker --predict_class_1 --predict_class_2 --predict_RT
+          nextflow run ${GITHUB_WORKSPACE} -profile test,docker
```

Comment:
I guess you don't need this anymore because it will do the same as the call above?

Reply:
ok right, I should change that back, forgot that we recently split it into two separate tests here